Pharmacy/PharamcyApi/Controllers/MedicineController.cs

```diff
@@ -82,6 +82,73 @@ public IActionResult GetById(int id)
             return Ok(medicine);
         }
 
+        [HttpGet]
```

Comment:
Since we have the same function here and bellow, why don't we extract that in a private method in the controller or a new function in the IMedicineReadRepository